### DISCUSSION TITLE
Our community is GROWING

### DISCUSSION THREAD

**Discussion by mgfuller333** [agencyenterprise/react-native-health#246] [General]
Happy to see that our weekly downloads have doubled over this year!!

This will be the most important React Native Library in existence! Keep the PRs coming! We have a lot of amazing updates to Apple HealthKit to cover with this Library!

Great work everyone! Join the Discord so we can collaborate with one another

**Comment by samuel-belcastro** [comment]
I tried to join the Discord link in the discussion, but the link was invalid

**Reply by ruan-azevedo** [reply]
Hey @samuel-belcastro I just created a new link, please try this one: https://discord.gg/d24g5XDePr
